dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtility.java

```diff
@@ -74,9 +65,34 @@ public class FeatureUtility {
     private static final String WEBSPHERE_LIBERTY_GROUP_ID = "com.ibm.websphere.appserver.features";
     private static final String BETA_EDITION = "EARLY_ACCESS";
     private static final String CONNECTION_FAILED_ERROR = "CWWKF1390E";
-    private String to;
-    private boolean isInstallServerFeature = false;
+    private static final VerifyOption DEFAULT_VERIFY = VerifyOption.enforce;
+    private static String to;
 
+    private boolean isInstallServerFeature = false;
+    private VerifyOption verifyOption;
+
+
+
+    protected FeatureUtility(InstallKernelMap map, File fromDir, List<File> esaFiles, Boolean noCache,
```

Comment:
add a comment here to let people know this constructor is for unit testing purposes and not to be used otherwise.